### ISSUE TITLE
Flashable ISO image for Raspberry Pis

### ISSUE THREAD

**Issue by rd18aac** [torch2424/live-stream-radio#107]
Would be nice if this project had a pre-configured flashable ISO image for Raspberry Pis and possibly other single board computers. Installation and setup is ok but a flashable image would be so convenient, especially since ffmpeg takes so long to install.

**Comment by torch2424** [comment]
Agreed. This would be rad 😄 Would take a bit of effort, if anyone is familiar with building pi images, feel free to send them my way 👍

**Comment by torch2424** [comment]
@karimamer Yay! You are the best 😄
